synapse/rest/client/room_batch.py

```diff
@@ -188,24 +188,26 @@ async def on_POST(
         assert_params_in_dict(body, ["state_events_at_start", "events"])
 
         assert request.args is not None
-        prev_events_from_query = parse_strings_from_args(request.args, "prev_event")
+        prev_event_ids_from_query = parse_strings_from_args(
+            request.args, "prev_event_id"
+        )
         chunk_id_from_query = parse_string(request, "chunk_id")
 
-        if prev_events_from_query is None:
+        if prev_event_ids_from_query is None:
```

Comment:
`parse_strings_from_args` has a `required` param to make this easy for you, but :shrug: